atm.js

```diff
@@ -1,21 +1,71 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+window.onload = function () {
+ $("div.account").addClass("zero");
+ };
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+var checkingBalance = parseFloat($("div#checking_balance").html().replace("$", ""));
+var savingsBalance = parseFloat($("div#savings_balance").html().replace("$", ""));
 
-}
+$("input[value='Deposit']").eq(0).on("click", function(){
+    var deposit = parseFloat($("input#checking_input").val());//Set their input equal to amount
+    checkingBalance += deposit;//Add amount to checkingBalance, i.e. checkingBalance += amount
+    $("div#checking_balance").html("$ " + checkingBalance.toFixed(2))//Update the "balance" div to reflect the new checkingBalance
+    if(checkingBalance > 0) {
+      $("div.account").eq(0).removeClass("zero");//Remove class zero
+    };
+})
```

Comment:
I think you might be repeating this code where we remove or add class zero based on balance. It might be better to abstract this into a function so you can just call the function any time you need to set classes.